Episerver/5.6.0/layerreferences/ref_languages.dcl

```diff
@@ -0,0 +1,6 @@
+data language english {
+    name = 'English'
+    short_name = 'en'
+    locale_id = 1033
+    enabled = true
```

Comment:
```suggestion
    short_name = 'en'
```
Better to rely on as few properties as necessary here to allow the definition to be changed slightly.